<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiStructureSelector</class>
 <widget class="QDialog" name="MultiStructureSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,0">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-size: 24pt;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Select Structures</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="refreshButton">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="acceptButton">
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTableView" name="structuresTable"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
